webapp/pages/settings/index.vue

```diff
@@ -111,13 +111,13 @@ export default {
     ...mapMutations({
       setCurrentUser: 'auth/SET_USER'
     }),
-    submit() {
-      this.sending = true
+    async submit() {
+      this.loadingData = true
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-17T14:09:33Z" title="Wednesday, April 17th 2019, 4:09:33 pm +02:00">Apr 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -111,13 +111,13 @@ export default {
     ...mapMutations({
       setCurrentUser: 'auth/SET_USER'
     }),
-    submit() {
-      this.sending = true
+    async submit() {
+      this.loadingData = true
```


This is a perfect opportunity to write a test.